packages/documentation/src/components/dashboard/DashboardDataFetch.jsx

```diff
@@ -1,16 +1,41 @@
+/* eslint-disable camelcase */
+import { Octokit } from '@octokit/rest';
+
+function getWorkflowRuns(repo) {
+  const octokit = new Octokit();
+
+  const params = {
+    owner: 'department-of-veterans-affairs',
+    repo,
+    workflow_id: 'continuous-integration.yml',
```

Comment:
This probably needs to be updated to include `manual-deploy-dev-staging.yml` since that workflow can also deploy commits.